starter-code/pizza.js

```diff
@@ -1 +1,47 @@
 // Write your Pizza Builder JavaScript in this file.
+$(document).ready(function(){
+
+
+  var pepperonniPrice =1;
+  var mushroomPrice =1;
+  var greenPrice =1;
+  var whiteSauce = 3;
+  var crustGluten = 5;
+  var total = 21;
+  $('.totalCost').html('$'+total);
+
+  $('.btn-pepperonni').click(function(){
+    $('.btn-pepperonni').toggleClass('active');
```

Comment:
When referencing the element for which you are writing and event handling function for, remember we can use the "this" keyword.

ex. $('.btn-pepperonni').click(function() {
     $(this).toggleClass('active');
.
.
.
}); //end click function